relayer/crates/starknet-chain-components/src/impls/queries/counterparty_connection_id.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
why there is an empty file? I think we remove this.

Reply:
Ah I wanted to implement the provider, but then figured we could reuse the same provider from Hermes SDK.